src/main/puzzle/piece/piece-types.gd

```diff
@@ -121,7 +123,8 @@ var piece_o := PieceType.new("o",
 		# color data
 		[[Vector2(10, 3), Vector2(6, 3), Vector2(9, 3), Vector2(5, 3)],
 		[Vector2(10, 3), Vector2(6, 3), Vector2(9, 3), Vector2(5, 3)]],
-		KICKS_NONE
+		KICKS_NONE,
+		[Vector2(0, 0), Vector2(0, 0), Vector2(0, 0), Vector2(0, 0)]
```

Comment:
These values seem like they could be calculated. Maybe something for refactoring day.